util/topsql/reporter/reporter.go

```diff
@@ -145,12 +146,15 @@ type SQLMeta struct {
 //
 // planBinaryDecoder is a decoding function which will be called asynchronously to decode the plan binary to string
 // MaxStatementsNum is the maximum SQL and plan number, which will restrict the memory usage of the internal LFU cache
-func NewRemoteTopSQLReporter(client ReportClient) *RemoteTopSQLReporter {
+func NewRemoteTopSQLReporter(clientRegistry *ReportClientRegistry, clients ...ReportClient) *RemoteTopSQLReporter {
```

Comment:
Why do we need to accept a registry as well as clients? It would be better to read clients from the registry?